### DISCUSSION TITLE
Please give me an intuition

### DISCUSSION THREAD

**Discussion by beginner984** [stuart-lab/signac#623] [Q&A] [answered]
Hello

I need your help please to get an intution

Cell-ranger atac pipeline returns two files **_filtered_peak_bc_matrix.h5_** and **_filtered_tf_bc_matrix.h5_**

You software uses **_Peak/Cell matrix_** by getting **_filtered_peak_bc_matrix.h5_**

What if I use **_filtered_tf_bc_matrix.h5_** file instead? 

Although, I tried that but I get this error

```
chrom_assay <- CreateChromatinAssay( + counts = counts, + sep = c(":", "-"), + genome = 'hg38', + fragments = 'fragments.tsv.gz', + min.cells = 10, + min.features = 200 + )
Error in .get_data_frame_col_as_numeric(df, granges_cols[["start"]]) : some values in the "start" column cannot be turned into numeric values In addition: Warning messages: 1: Expected 3 pieces. Additional pieces discarded in 1 rows [67]. 2: Expected 3 pieces. Missing pieces filled withNA in 534 rows [1, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 15, 16, 17, 18, 19, 20, 21, 22, 24, ...].
```

My question please is, what is the application of **_filtered_tf_bc_matrix.h5_** in your software? Is your sofware able to get this file and use that as ArchR software to `TSS `enrichment?

Thank you so much for any help

**Comment by timoast** [comment, marked as answer]
We do not typically use the 10x Genomics TF matrix. The error you see is because the features in that matrix are not genomic coordinates, which is a requirement of `CreateChromatinAssay`. See https://github.com/timoast/signac/issues/620

**Comment by beginner984** [comment]
Sorry, then how I can do TSS enrichment by your software? Which input?

**Reply by timoast** [reply]
Use the `TSSEnrichment()` function. Please see documentation that is available on the website: https://satijalab.org/signac/
